src/fragments/lib/logging/ios/setup_logging/30_initialize_without_config.mdx

```diff
@@ -0,0 +1,26 @@
+To use the Amplify Logger and Amplify Auth categories in your app, you need to create and configure their corresponding plugins by calling the `Amplify.add(plugin:)` and `Amplify.configure()` methods.
+
+Add the following imports to the top of your main `App` file:
+
+```swift
+import Amplify
+import AWSCognitoAuthPlugin
+import AWSCloudWatchLoggingPlugin
+```
+
+Add the following code to its initializer. If there is none, you can create a default `init`.  The `<log-group-name>` and `<region>` are values you specified in the CDK construct as part of provisioning your backend resources.  These values can also be found at the end of the output logs when deploying the sample CDK construct. 
```

Comment:
add the following code to its initializer?

What does that mean? should we say Add the following code to your app's initializer?

Reply:
its refers to the main app entry point/initializer.  We use the same verbiage in all other existing code snippets in other categories, so I'd like to keep it consistent.